core/modules/comment/comment.install

```diff
@@ -364,6 +364,75 @@ function comment_update_1003() {
 }
 
 /**
+ * Replace any "Recent comments" blocks previously provided by comment.module
+ * with blocks provided by block displays of the "Recent comments" view.
+ */
+function comment_update_1004() {
+
+  // Enable the "Recent comments" view if not already enabled.
+  $view = views_get_view('comments_recent');
+  if (!views_view_is_enabled($view)) {
```

Comment:
We can't count on public APIs like this being available in update hooks, this will need to be written through config() calls to check if the view exists and is enabled.